internal/controller/cilium_flow_collector.go

```diff
@@ -25,48 +24,62 @@ type CiliumFlowCollector struct {
 
 const (
 	ciliumHubbleRelayMaxFlowCount uint64 = 100
-	ciliumHubbleRelayServiceName  string = "hubble-relay"
-)
 
-var (
-	ErrHubbleNotFound   = errors.New("hubble Relay service not found; disabling Cilium flow collection")
-	ErrNoPortsAvailable = errors.New("hubble Relay service has no ports; disabling Cilium flow collection")
+	// Constants for fetching the mTLS secret
+	hubbleMTLSSecretName       string = "hubble-relay-client-certs"
+	ciliumHubbleRelayNamespace string = "kube-system"
```

Comment:
To be consisent, always refer to "Hubble" as "Cilium Hubble"?
```suggestion
	ciliumHubbleMTLSSecretName string = "hubble-relay-client-certs"
	ciliumHubbleRelayNamespace string = "kube-system"
```